hpcbench/benchmark/ior.py

```diff
@@ -226,21 +237,26 @@ def execution_matrix(self, context):
                 yield command
 
     def _execution_matrix(self, api):
+        if self.path:
+            if '-F' not in self.options:
+                opath = ['-o', self.path+'/data']
```

Comment:
Prefer using ` os.path.join` to hard-coded path separator.
Besides, the portable `/` is `os.sep`